internal/docstore/driver/driver.go

```diff
@@ -90,11 +90,10 @@ type ActionListError []struct {
 // A Query defines a query operation to find documents within a collection based
 // on a set of requirements.
 type Query struct {
-	FieldPaths [][]string       // the selected field paths
-	Filters    []Filter         // a list of filter for the query
-	Limit      int              // the number of result in one query request
-	StartAt    interface{}      // define the starting point of the query
-	Iter       DocumentIterator // the iterator for Get action
+	FieldPaths [][]string  // the selected field paths
```

Comment:
IMHO these comments need more details.

E.g.,
* for `FieldPath`, add something like: "The returned Documents should only have these fields."
* for `Filters`, add that the filters should be ANDed together
* for `Limit`, is the driver allowed to return `EOF` before `Limit` is reached even if more are available?